src/tests/ftest/container/snapshot.py

```diff
@@ -307,6 +307,35 @@ def test_snapshot_negativecases(self):
                 self.fail(
                     "##(7.1)Expecting error RC: -1002  did not show.")
 
+    def disaply_snapshot_test_data(self, test_data, ss_index):
+        """Display the snapshot test data.
+
+        Args:
+            test_data: list of dictionary snapshot testdata keys:
+                coh: container handler
+                container_epoch: container epoch
+                snapshot:      snapshot handle
+                tst_obj:       test object
+                tst_data:      test data
+            ss_index: snapshot index to be displayed.
+        """
+        if len(test_data) < ss_index - 1:
+            self.log.info("##Under to display test_data info, "
+                          "index out of range.")
+        else:	
```

Comment:
(style)  trailing whitespace

Reply:
Plese see the new script commit for the fixes.

Reply:
Plese see the new script commit for the fixes.